### PR TITLE
DTExchange 8.2.0 version  support

### PR DESCRIPTION
**Pull request by 2energycell**

### CODE REVIEW

**Comment by 2energycell** [comment]
@joshh-devrel please approve this **ASAP**. The DTExchange 8.2.0 is already live, as well as Google adapter for it.
But it lacks a crucial change.
Please be noted this is our new official fork, the previous Fyber Engineering account is not relevant anymore.
@TomerFyber FYI

File: adapters/DTExchange/DTExchangeAdapter/GADMAdapterFyberUtils.m [hunk 1]
```diff
@@ -72,7 +72,7 @@ GADVersionNumber GADMAdapterFyberVersionFromString(NSString *_Nonnull versionStr
     builder.spotID = spotID;
     builder.timeout = 10;
     builder.userData = extras.userData;
-    builder.muteAudio = extras.muteAudio;
+    IASDKCore.sharedInstance.muteAudio = extras.muteAudio;
```
Comment (joshh-devrel) [inline adapters/DTExchange/DTExchangeAdapter/GADMAdapterFyberUtils.m:75]: @2energycell 

Since this is not related to the builder, we can move it before building the request (i.e. line 69).

Also, can you confirm if the below check is needed? Publishers might not have set the `muteAudio`.
```
  if (extras.muteAudio) {
    IASDKCore.sharedInstance.muteAudio = extras.muteAudio;
  }
```

Comment (2energycell) [reply adapters/DTExchange/DTExchangeAdapter/GADMAdapterFyberUtils.m:75]: correct; but I will submit one more PR shortly; it covers 2 previous PRs;

Comment (2energycell) [reply adapters/DTExchange/DTExchangeAdapter/GADMAdapterFyberUtils.m:75]: @joshh-devrel this PR https://github.com/googleads/googleads-mobile-ios-mediation/pull/409
can be merged instead of 8.2.0 and 8.2.1 and it supports 8.2.0 SDK as well; thx;

**Comment by 2energycell** [comment]
closing this PR; please merge this one instead: https://github.com/googleads/googleads-mobile-ios-mediation/pull/409
